sn_node/src/event.rs

```diff
@@ -71,8 +71,8 @@ pub enum NodeEvent {
     TransferNotif {
         /// Public key the transfer notification is about
         key: PublicKey,
-        /// The encrypted transfer
-        transfer: Transfer,
+        /// The cash notes
```

Comment:
Just curious, so this means the cash_notes are no longer encrypted, right?
is it ok?

Reply:
correct, we can change this later to encrypt them here too, anyhow we only send royalties payment cash notes on these notifications now